lib/wallaroo/core/topology/partition.pony

```diff
@@ -19,6 +19,7 @@ Copyright 2017 The Wallaroo Authors.
 use "collections"
```

Comment:
Let's rename this file `partitions.pony`

Reply:
done